### PR TITLE
chore: Bump tests coverage

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: test/CSAccounting.t.sol [hunk 1]
```diff
@@ -108,6 +108,43 @@ contract CSAccountingConstructorTest is CSAccountingBaseConstructorTest {
             365 days
         );
     }
+
+    function test_constructor_RevertWhen_InvalidBondLockRetentionPeriod_Condition1()
```
Comment (skhomuti) [inline test/CSAccounting.t.sol:112]:
```suggestion
    function test_constructor_RevertWhen_InvalidBondLockRetentionPeriod_MinMoreThanMax()
```

File: test/CSAccounting.t.sol [hunk 2]
```diff
@@ -108,6 +108,43 @@ contract CSAccountingConstructorTest is CSAccountingBaseConstructorTest {
...
+        accounting = new CSAccounting(
+            address(locator),
+            address(0),
+            10,
+            4 weeks,
+            2 weeks
+        );
+    }
+
+    function test_constructor_RevertWhen_InvalidBondLockRetentionPeriod_Condition2()
```
Comment (skhomuti) [inline test/CSAccounting.t.sol:125]:
```suggestion
    function test_constructor_RevertWhen_InvalidBondLockRetentionPeriod_MaxTooBig()
```

File: test/CSAccounting.t.sol [hunk 3]
```diff
@@ -108,6 +108,43 @@ contract CSAccountingConstructorTest is CSAccountingBaseConstructorTest {
...
+        public
+    {
+        vm.expectRevert(CSBondLock.InvalidBondLockRetentionPeriod.selector);
+        accounting = new CSAccounting(
+            address(locator),
+            address(0),
+            10,
+            4 weeks,
+            uint256(type(uint128).max) + 1
```
Comment (skhomuti) [inline test/CSAccounting.t.sol:134]:
```suggestion
            uint256(type(uint64).max) + 1
```

File: test/SigningKeys.t.sol [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+    }
+
+    function testFuzz_saveKeysSigs(
+        uint256 nodeOperatorId,
+        uint256 startIndex,
+        uint256 keysCount
+    ) public {
+        vm.assume(keysCount > 0);
+        vm.assume(keysCount < 100);
+        unchecked {
```
Comment (skhomuti) [inline test/SigningKeys.t.sol:88]: I don't think if it makes sense in tests

File: test/SigningKeys.t.sol [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+            vm.assume(startIndex + keysCount > startIndex);
```
Comment (skhomuti) [inline test/SigningKeys.t.sol:89]: it looks like `keysCount > 0`, which is above already

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: test/SigningKeys.t.sol (hunk 4 above)
Comment (dgusakov) [reply test/SigningKeys.t.sol:88]: It does. It is an overflow check. Take a closer look

File: test/SigningKeys.t.sol (hunk 5 above)
Comment (dgusakov) [reply test/SigningKeys.t.sol:89]: This is an overflow check for sum

**Review by skhomuti — APPROVED** [review]

**Review by skhomuti — DISMISSED** [review]

File: test/SigningKeys.t.sol [hunk 6]
```diff
@@ -0,0 +1,594 @@
...
+        uint64 nodeOperatorId,
+        uint32 startIndex,
+        uint32 keysCount
+    ) public {
+        vm.assume(keysCount > 0);
+        vm.assume(keysCount < 100);
+        unchecked {
+            vm.assume(startIndex + keysCount > startIndex);
+        }
+        vm.assume(startIndex + keysCount < type(uint32).max);
```
Comment (madlabman) [inline test/SigningKeys.t.sol:204]: It seems it's covered by the line 202.

File: test/SigningKeys.t.sol [hunk 7]
```diff
@@ -0,0 +1,594 @@
...
+        );
+        uint256 newKeysCount = signingKeys.saveKeysSigs(
+            nodeOperatorId,
+            startIndex,
+            keysCount,
+            pubkeys,
+            signatures
+        );
+
+        assertEq(newKeysCount, startIndex + keysCount);
```
Comment (madlabman) [inline test/SigningKeys.t.sol:218]: I would add a check that `loadKeys` returns saved ones.

File: test/SigningKeys.t.sol [hunk 8]
```diff
@@ -0,0 +1,594 @@
...
+        vm.expectRevert(SigningKeys.InvalidKeysCount.selector);
+        signingKeys.removeKeysSigs(
+            nodeOperatorId,
+            startIndex,
+            keysCount,
+            startIndex + keysCount - 1
+        );
+    }
+
+    function test_removeKeysSigs_revertWhen_totalKeysToHigh() public {
```
Comment (madlabman) [inline test/SigningKeys.t.sol:327]:
```suggestion
    function test_removeKeysSigs_revertWhen_totalKeysTooHigh() public {
```

**Review by vgorkavenko — DISMISSED** [review]

File: test/SigningKeys.t.sol (hunk 7 above)
Comment (dgusakov) [reply test/SigningKeys.t.sol:224]: added

**Review by skhomuti — APPROVED** [review]

**Review by madlabman — APPROVED** [review]

**Pull request merged**